.changeset/empty-birds-beam.md

```diff
@@ -0,0 +1,8 @@
+---
+"uploadthing": patch
```

Comment:
I would probably call this a minor. In my mind, we should generally try to version based on:
patch --> bugfix, style changes, or other very small changes 
minor --> "typical" release (new features)
major --> breaking changes

Reply:
Agreed, also added docs based on this being a minor. Missed that 🤩